src/vivarium_public_health/metrics/disability.py

```diff
@@ -120,16 +121,27 @@ def disability_weight_aggregator(self, dw: pd.DataFrame) -> float:
 
     def report(
```

Comment:
I think it's a bit confusing to call this method `report`, since it's not actually the method being sent as the report function when you register observations. Same thing in your other observer.

Reply:
Not sure I understand. In line 100, we register the observation with `report=partial(self.report, cause_state)`. The self.report is indeed this `report` method defined here.